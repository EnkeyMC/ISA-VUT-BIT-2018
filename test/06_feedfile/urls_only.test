NAME=Only URLs
ARGS=-f resources/feedfile_urls_only.txt
RC=0
CHECK_OUTPUT=1
