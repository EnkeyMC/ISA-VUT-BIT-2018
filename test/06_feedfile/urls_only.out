*** xkcd.com ***
I'm a Car

*** xkcd.com ***
I'm a Car

*** xkcd.com ***
I'm a Car

*** xkcd.com ***
I'm a Car
